source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<p>I added the tags, including <code>library/dask-databricks</code>and <code>library/dask-deltatable</code> but I was wondering whether we want to document these two packages in the <code>Tools/Packages</code> on the <a href="https://docs.rapids.ai/deployment/stable/tools/" target="_blank">deployment page</a>?      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think we cover `dask-databricks` enough on the Databricks platform page, if we made a tools page it would be much of the same content. I don't think we need to document `dask-deltatable` because it's not a deployment tool.